noir-projects/noir-contracts/contracts/lending_contract/src/main.nr

```diff
@@ -81,7 +80,7 @@ contract Lending {
 
             // accumulator *= multiplier, and multiplier >= 1
             asset.interest_accumulator = (asset.interest_accumulator * multiplier) / precision;
-            asset.last_updated_ts = context.timestamp();
+            asset.last_updated_ts = timestamp;
```

Comment:
Sneaked in this random change. It bothered me that we don't use timestamp here when we assigned it as a local variable above.